types/react-map-gl/index.d.ts

```diff
@@ -1,7 +1,8 @@
-// Type definitions for react-map-gl 4.0
+// Type definitions for react-map-gl 4.1
 // Project: https://github.com/uber/react-map-gl#readme
 // Definitions by: Robert Imig <https://github.com/rimig>
 //                 Fabio Berta <https://github.com/fnberta>
+//                 Sander Siim <https://github.com/sandersiim>
```

Comment:
This will have the effect that you will get notified on every issue and PR related to react-map-gl types. Just something to be aware of.